### PR TITLE
17402 complete status endpoint

### PR DESCRIPTION
**Pull request by hschink**
[`* `#17402` Complete status endpoint`](https://community.openproject.org/work_packages/17402)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://fix-17402-complete-status-endpoint.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: doc/apiv3-documentation.apib [hunk 1]
```diff
@@ -1223,8 +1223,6 @@ The request body is the actual string that shall be rendered as HTML string.
 | isDefault|             | Boolean |             | true    |                 READ |
 | isClosed | are tickets of this status considered closed? | Boolean | | true | READ |
 | defaultDoneRatio | The percentageDone being applied when changing to this status | Integer | 0 <= x <= 100 | 50 | READ |
-| createdAt|             | Timestamp | Returned in ISO 8601 format - YYYY-MM-DDTHH:MM:SSZ | | READ |
-| updatedAt|             | Timestamp | Returned in ISO 8601 format - YYYY-MM-DDTHH:MM:SSZ | | READ |
```
Comment (NobodysNightmare) [inline doc/apiv3-documentation.apib:1227]: Why did you drop that?
~~It did exist in APIv2 and it is actually part of the model, isn't it?~~ (apparently am wrong here)

The vision (as far as I am concerned), is that _some day(tm)_ all aspects of OpenProject are available over the API... so I do not see the reason to spare out arbitrary parts...

edit: Seems that what I am saying is not true and a status does not have this information

Comment (hschink) [inline doc/apiv3-documentation.apib:1227]: @NobodysNightmare Please refer to https://github.com/opf/openproject/commit/9c15745312cc23f789dfa811fbc859f2c56e0d99.

**Comment by myabc** [comment]
Looks good. :+1:

**Pull request merged**